docs/community/governance.md

```diff
@@ -20,16 +20,17 @@ Established members are expected to demonstrate their adherence to the principle
 
 The current team members are:
 
-- Cyril Tovena                  [cyriltovena](https://github.com/cyriltovena)
-- David Kaltschmidt             [davkal](https://github.com/davkal)
-- Edward Welch                  [slim-bean](https://github.com/slim-bean)
-- Goutham Veeramachaneni        [gouthamve](https://github.com/gouthamve)
-- Joe Elliott                   [joe-elliott](https://github.com/joe-elliott)
-- Owen Diehl                    [owen-d](https://github.com/owen-d)
-- Robert Fratto                 [rfratto](https://github.com/rfratto)
-- Sandeep Sukhani               [sandeepsukhani](https://github.com/sandeepsukhani)
-- Tom Braack                    [sh0rez](https://github.com/sh0rez)
-- Tom Wilkie                    [tomwilkie](https://github.com/tomwilkie)
+- Aditya C S                         [adityacs](https://github.com/adityacs)
```

Comment:
Should we add name of the company where Aditya works?

Reply:
Cyril had initially added it. I asked him to remove. The thing is, since I work on this project only in my free time, I don't think I am officially allowed to have my company name here.